src/odemis/driver/test/tmcm_test.py

```diff
@@ -585,14 +585,16 @@ class TestCanActuator(unittest.TestCase):
 
     def setUp(self):
         if TEST_NOHW:
-            address = "fake"
+            channel = "fake"
         else:
-            address = "can0"
+            channel = "can0"
 
         current_dir = os.path.abspath(os.path.dirname(__file__))
-        parent_dir = os.path.abspath(current_dir + "/../")
-        datasheet = os.path.join(parent_dir, 'TMCM-1240_CANopen_V322.dcf')
-        self.dev = tmcm.CANController("Can Actuator", None, address, 1, datasheet, ['x'], refproc=tmcm.REFPROC_STD)
+        datasheet = os.path.join(current_dir, "..", 'TMCM-1240_CANopen_V322.dcf')
+        # 200 steps / cycle and 2 ** 8 µsteps per step
+        sz = [2 * math.pi / (200 * 2 ** 8)]  # µstep size in rad
+        self.dev = tmcm.CANController("Can Actuator", None, channel, 1, datasheet, ['x'], refproc="Standard",
+                                      param_file=ABS_PATH + "/tmcm-pd1240.tmcm.tsv", ustepsize=sz)
```

Comment:
Please pass put the name of every parameter. Otherwise it's not very clear what they mean. (eg: 1). Also, please add `rng` and `unit`, at least for the sake of code coverage.